### ISSUE TITLE
What does Do not disturb mean?

### ISSUE THREAD

**Issue by dodalovicgran** [pop-os/pop#2863]
Hi, there's this setting:
![image](https://user-images.githubusercontent.com/116893230/232154683-6c687853-0807-4f6f-bcab-e8d0f21cdb85.png)

Although it's turned on, I'm still getting notifications. This confuses me. 

Thank You

**Comment by dodalovicgran** [comment]
So, is it designed to be broken or?

**Comment by leviport** [comment]
>So, is it designed to be broken

No, it is not "designed to be broken". The kinds of notifications you still receive will be helpful. It suppresses `notify-send` notifications on my end, so whichever notifications you're seeing might be a different type, coming from a different application.

**Comment by dodalovic** [comment]
Ok
It's far from ideal that user is unable to decide that under some circumstances it's absolutely not acceptable to get anything shown.

Like, I have to hold a webinar and I'm most likely to get some rubbish shown on the screen, 

There needs to be a Don't disturb ON/OFF toggle, and additionally when it's on there's a su toggle to allow only li

**Comment by leviport** [comment]
Yeah, I understand that. I'm asking what applications are sending notifications that are not blocked by DnD mode. Like I said, `send-notify`, which is a way to test notifications in Gnome, does not display notifications on my machine while DnD mode is active. If you can tell us the names of applications that are _not_ blocked by DnD mode, it would be extremely helpful. If we can confirm that behavior, we can refine and reopen this issue.

**Comment by dodalovicgran** [comment]
I have no idea? Browser?

**Comment by leviport** [comment]
In order for us to understand the nature of the bug, we'll have to first be able to understand the nature of the notifications. If you have more info later, please share it with us so that we can investigate effectively.

**Comment by dodalovicgran** [comment]
What is the "nature of the notifications"? I believe that Google Calendar sends me something, Reddit comment or what not...

**Comment by leviport** [comment]
We need exact details. What browser? What site? What happened that generated the notification?

**Comment by dodalovicgran** [comment]
Chrome or Edge
It comes from Google Calendar I think. 
Or even if someone comments on a Reddit post or similar.

**Comment by jacobgkau** [comment]
There is an upstream issue in GNOME regarding Chrome/Chromium-based browser notifications: https://gitlab.gnome.org/GNOME/gnome-shell/-/issues/2148 

We can reopen this issue for tracking purposes and rename it to reflect the issue you were trying to report, but the issue will probably need to be fixed upstream.
